.github/workflows/fmt.yml

```diff
@@ -6,21 +6,19 @@ on:
   pull_request:
   workflow_dispatch:
 
+env:
+  RUST_NIGHTLY_VERSION: 2024-03-14 # 1.78.0-nightly
```

Comment:
Could we please put these env variables into one file? So we don't always need to touch all the files. We can then also relax the review requirements on this file to the same level as the normal source files.

Reply:
Ty!